### ISSUE TITLE
Are The Projects Listed In The Readme Receiving Any Financial Support From Your Grant/Clarification About Grant Funds And Project?

### ISSUE THREAD

**Issue by bonedaddy** [bneiluj/covid-19-open-source-ventilators#3]
So I've been following a lot of the work being done for open-source ventillators, and I see that @bneiluj and someone else created a grant on gitcoin https://gitcoin.co/grants/578/open-source-covid-ventilator-oscv?tab=description

Naturally because this is an area of development I'm following I was very curious. Unfortunately it seems like you're taking the work done by other people, and using it as reason for creating a grant and financial gain. At least on the surface it is very hard to deduce what the money is actually going to, who its going to, and who did what work.

I noticed [this website](https://www.isinnova.it/easy-covid19-eng/) which I saw linked to on @bneiluj twitter, claiming "non profit". Is this actually the case? Can anyone actually trust this is not for profit? I'm sure everyone is well aware that non-profit organizations have and do take advantage of the non-profit status for financial gain. Furthermore ISINNOVA doesn't even appear to be a publicy registered non-profit entity, which makes the supposedly "non-profit claims" a little dubious.

Without any clarification this seems like top-tier shadiness and slimyness being disguised as a public good for financial gain. Please provide some clarifications on all of this because it seems like you're taking advantage of the hardwork created by other people without giving back to them. Just because its open-source, doesn't mean you shouldn't give back in some way.

edit:

Furthermore I found [this tweet](https://twitter.com/bneiluj/status/1243557173759746048) which seems a bit concerning. This tweet on the surface seems like you're asking people to help out by donating their time, financial resources, and 3D printing equipment/skills to the cause.

Will these people also be receiving the funds you two get? I understand 3D printing and all that costs money, but if you're asking other people to help out and its not just you two, certainly they should be included in the grant.

**Comment by bneiluj** [comment]
Hey @bonedaddy - it's non profit. The donations on gitcoin are used to buy decathlon masks and filaments for people that can't afford to buy them. 
It's a worldwide movement, people can join the initiative started by https://www.isinnova.it/easy-covid19-eng/ as mentioned on my tweets and many other posts.

**Comment by bonedaddy** [comment]
Thanks for the response.  I did read the gitcoin readme, and it doesn't mention that its not for profit, nor does it mention that you are giving grant funds to people who participate that cant afford the equipment. The only not for profit mention is on isinnova's website, which I don't think is a part of this? You're just using their design.

It's a worldwide movement yes, but you are on twitter asking people to get involved, but the grant is only listed as giving money to you and one other person. The github readme doesn't mention anything about buying filaments for people who who cant afford them, nor does it mention anything about a payment setup that will be used to pay other people who are donating their resources.

It very much seems like the money is going to you and one other person, as opposed to everyone involved, and your response doesn't do much to clear this up. Maybe if the readme on gitcoin clearly indicated how other people getting involved can get some funds from the grant but it doesn't say that at all.

And I strongly disagree that everything is transparent. I've looked on the telegram channel and I see nothing about the funds being available to anyone other than you and one other person. It doesn't say that on twitter either, and it doesn't say that on gitcoin.

The only time that has been said is now, in your response. The gitcoin hasn't even been updated since your response to list that its not for profit, and that other people involved can get access to the grant funds.
